libddwaf.gemspec

```diff
@@ -27,13 +27,7 @@ Gem::Specification.new do |spec|
   end
 
   spec.files =
-    `git ls-files -z`
-    .split("\x0")
-    .reject { |f| f.match(%r{^(spec|[.]circleci)/}) }
-    .reject do |f|
-      ['.dockerignore', '.env', '.rspec', '.rubocop.yml', '.rubocop_todo.yml',
-      '.simplecov', 'Gemfile', 'Rakefile', 'docker-compose.yml'].include?(f)
-    end
+    `git ls-files -z`.split("\x0") & (Dir['lib/**/*.rb'] + Dir['sig/**/*.rbs'] + ['NOTICE', 'CHANGELOG.md'] + Dir['LICENSE*'])
```

Comment:
The end result is:
```
CHANGELOG.md
LICENSE
LICENSE-3rdparty.csv
LICENSE.Apache
LICENSE.BSD3
NOTICE
lib/datadog/appsec/waf.rb
lib/datadog/appsec/waf/version.rb
lib/libddwaf.rb
sig/datadog/appsec/waf.rbs
sig/datadog/appsec/waf/version.rbs
sig/libddwaf.rbs
```

Not going to lie it was a little hard for me to understand fully, so I had to run it 😄